components/sprayproxy/production/kustomization.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=c6f8bf26007c140d51326760e16eb52ae5072e3b
+  - https://github.com/konflux-ci/sprayproxy/config?ref=53ae76d32a32be7babf32df376b89d0623e78726
   - pipelines-as-code-secret.yaml
 
 images:
-  - name: ko://github.com/redhat-appstudio/sprayproxy
+  - name: ko://github.com/konflux-ci/sprayproxy
     newName: quay.io/redhat-appstudio/sprayproxy
```

Comment:
Is it possible newName must also use new org name?  The pod cannot start on staging: Back-off pulling image
            "quay.io/redhat-appstudio/sprayproxy:53ae76d32a32be7babf32df376b89d0623e78726"

Reply:
the image doesnt exist yet, with this sha, I am trying to ask folks how images re built for sprayproxy

Reply:
the repo is missing the github app and hence the ci is not running and image not created
Asking on forum-konflux-devlopper if someone can add the github app